src/plugin.js

```diff
@@ -39,6 +39,7 @@ class Flash extends Tech {
   */
   constructor(options, ready) {
     super(options, ready);
+    this.player_ = videojs(options.playerId);
```

Comment:
we shouldn't reference the player inside of techs if we can at all avoid it. We try and decouple the player and the tech as much as we can.